acceptance/variables/variables.go

```diff
@@ -1,14 +1,53 @@
 package variables
 
-const (
-	ContainerBaseImage = "ubuntu:bionic"
-	DockerfileName     = "Dockerfile"
-	VolumeHelperImage  = "busybox"
-)
+type VariableHelper struct {
```

Comment:
In the exec.d PR I decided to move this os-specific logic into the `acceptance` package: https://github.com/buildpacks/lifecycle/blob/11ca8ef99c1626fa34ea087e74993d8746c7cf61/acceptance/variables_unix.go

I don't know if it was the right call. I'm happy to embrace this pattern instead...

Reply:
Good call. I replaced `VariableHelper` with `const`s in `variables.go` (and a `ctrPath()` function for some simple path conversion)